lib/rules/cli/4.1.yaml

```diff
@@ -397,6 +397,19 @@
     :l: -l <value>
     :resource-names-file: --resource-names-file=<value>
     :svc_name: <value>
+:image_mirror:
+  :cmd: oc image mirror
```

Comment:
maybe it's better    :cmd: oc image mirror <src> <des>
other LGTM

Reply:
@wewang58 Updated, thanks